doc/source/tune/doc_code/faq.py

```diff
@@ -439,3 +439,62 @@ def train_func(config):
     )
     tuner.fit()
     # __no_chdir_end__
+
+
+# __iter_experimentation_initial_start__
+from ray import air, tune
+from ray.air import Checkpoint, session
+import random
+
+
+def trainable(config):
+    for epoch in range(1, config["num_epochs"]):
+        # Do some training...
+
+        session.report(
+            {"score": random.random()},
+            checkpoint=Checkpoint.from_dict({"model_state_dict": {"x": 1}}),
+        )
+
+
+tuner = tune.Tuner(
+    trainable,
+    param_space={"num_epochs": 10, "hyperparam": tune.grid_search([1, 2, 3])},
+    tune_config=tune.TuneConfig(metric="score", mode="max"),
+)
+result_grid = tuner.fit()
+
+best_result = result_grid.get_best_result()
+best_checkpoint_path = best_result.checkpoint.uri
+# __iter_experimentation_initial_end__
+
+
+# __iter_experimentation_resume_start__
+def trainable(config):
+    # Add logic to handle the initial checkpoint.
+    checkpoint_path = config["start_from_checkpoint"]
+    checkpoint = Checkpoint.from_uri(checkpoint_path)
+    model_state_dict = checkpoint.to_dict()["model_state_dict"]
+    # Initialize a model from the checkpoint...
+
+    for epoch in range(1, config["num_epochs"]):
+        # Do some training...
+
+        session.report(
+            {"score": random.random()},
+            checkpoint=Checkpoint.from_dict({"model_state_dict": {"x": 1}}),
+        )
+
+
+new_tuner = tune.Tuner(
+    trainable,
+    param_space={
+        "num_epochs": 10,
+        "hyperparam": tune.grid_search([4, 5, 6]),
+        # Use the `best_checkpoint_uri` from above.
+        "start_from_checkpoint": best_checkpoint_path,
```

Comment:
Nope, this is just some random config name for users to seed the new run with a checkpoint. This actually only works if the checkpoint is available in s3. Otherwise, if it's a local path, some nodes will not find it.